src/components/button/styled.tsx

```diff
@@ -2,127 +2,82 @@ import React from "react"
 import styled, { css } from "styled-components"
 import { CircularProgress } from "@rmwc/circular-progress"
 import { getColor } from "../../theme/utils"
-import { ButtonProps, ButtonType } from "./button"
+import { ButtonProps } from "./button"
 import { Icon } from "../icon"
+import { DEFAULT, NO_FILL, BORDER_LESS } from "./constants"
+
+const getGreenHaze = getColor(["green", "greenHaze"])
+const getWhitePure = getColor(["white", "pure"])
+const getGreenMalachite = getColor(["green", "malachite"])
 
 const buttonPropsMap = new Map<string, (props: ButtonProps) => any>([
-  [
-    "divWidth",
-    props => {
-      return props.label ? "134px" : "46px"
-    },
-  ],
+  ["divWidth", ({ label }) => (label ? "134px" : "46px")],
```

Comment:
px should be removed in favour of getSizeBy / other means of getting size from theme-defined constant.